tests/table.spec.ts

```diff
@@ -95,15 +95,16 @@ column_header_group = column_header_group.layout_hints(column_groups=column_grou
   await expect(page.locator('.iris-grid-panel .iris-grid')).toHaveScreenshot();
 });
 test.describe('tests complex table operations', () => {
-  test('can select distinct values', async ({ page }) => {
+  let page: Page;
+
+  test.beforeEach(async ({ browser }) => {
+    page = await browser.newPage();
```

Comment:
I like the idea of separating it into a new file but just to prevent the headache of merge conflicts I'm going to do that in the next batch of e2e PRs I have coming up. Just going to add the afterEach method for now